Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:2.7
```

Comment:
I'm not sure we want to do this. We do most of our testing inside Ubuntu. Part of the point of Dockerization is to make debugging for us easier, and adding another OS/platform as an execution engine has testing costs.

Also, the python image, while perhaps saving some time on building prerequisites, doesn't save any space.

Image | version  | size
---- | ----  | ----
nop-image|latest|726MB
python-image|latest|946MB

Reply:
Python image is based on Debian, so I do not see any problem here. One should not reinvent the wheel, and if you need Ubuntu/Debian + Python then you should directly use the Python image.

Reply:
Yeah, I'm not worried about the size at all, except to say that there is more stuff included with library/python. What I'm worried about is reproducibility of results.

I'm willing to try it both ways (ubuntu and python), but using Debian may stress out install scripts a little. I'm trying to keep Debian operational, if for no other reason than the free Google Cloud Shell has a Debian environment.